File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        unq = [nums[0]]
        for i in range(1,len(nums)):
            if nums[i] != nums[i-1]:
                unq.append(nums[i])
        ans = len(unq)
        for i in range(0,ans):
            nums[i] = unq[i]

        return ans        

File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) 
    {
        vector<int> unq;
        unq.push_back(nums[0]);
        int n = nums.size();

        for (int i=1; i < n ;i++)
        {
            if ( nums[i] != nums[i-1]) unq.push_back(nums[i]);
        }
        int ans = unq.size();
        for(int i=0 ;i < ans ;i++)
        {
            nums[i] = unq[i];
        }

        return ans;
        
        
    }
};